agimus_controller_ros/agimus_controller.py

```diff
@@ -21,8 +24,8 @@ def update_state_machine(self):
             if self.elapsed_time is not None:
                 self.change_state()
             return
-        if (self.elapsed_time < 0.05 and self.last_elapsed_time >= 0.05) or (
-            self.elapsed_time >= 0.05 and self.last_elapsed_time < 0.05
+        if (self.elapsed_time < 0.1 and self.last_elapsed_time >= 0.1) or (
+            self.elapsed_time >= 0.1 and self.last_elapsed_time < 0.1
```

Comment:
To be parametrized

Reply:
to be honest I'm a bit suprised that 0.05 was not enough but in the simulations and expes I did since this modification, it was working properly